src/braket/default_simulator/density_matrix_simulator.py

```diff
@@ -120,9 +213,21 @@ def properties(self) -> GateModelSimulatorDeviceCapabilities:
                             {"name": "Probability", "minShots": 0, "maxShots": max_shots},
                             {"name": "DensityMatrix", "minShots": 0, "maxShots": 0},
                         ],
-                    }
+                    },
                 },
                 "paradigm": {"qubitCount": qubit_count},
                 "deviceParameters": GateModelSimulatorDeviceParameters.schema(),
             }
         )
+
+    @singledispatchmethod
+    def _run_internal(
+        self, ir: Union[OQ3Program, JaqcdProgram], *args, **kwargs
```

Comment:
The OQ branch should only accept `OQ3Program`, not `JaqcdProgram`

Reply:
Will fix with the default change